pypsa/io.py

```diff
@@ -904,6 +904,8 @@ def import_components_from_dataframe(network, dataframe, cls_name):
         else:
             if static_attrs.at[k, "type"] == "string":
                 dataframe[k] = dataframe[k].replace({np.nan: ""})
+            if static_attrs.at[k, "type"] in ["float", "int"]:
+                dataframe[k] = dataframe[k].replace({np.nan: 0})
```

Comment:
```suggestion
                dataframe[k] = dataframe[k].fillna(0)
```

Reply:
Just tried to stick to the old convention, if we use this one then I'd suggest to use `dataframe[k] = dataframe[k].fillna("")` as well (1-2 lines earlier)